packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -113,6 +115,17 @@ const VisitHeader: React.FC = () => {
   const navMenuItems = useAssignedExtensions('patient-chart-dashboard-slot').map((extension) => extension.id);
   const { startVisitLabel } = useConfig();
 
+  // I thought this would tell when a form is updated
+  // const globalFormStore = getGlobalStore('ampath-form-state', {}).getState();
```

Comment:
So, if we need to be informed by the form engine whether there are changes, that change needs to be part of the form engine or the wrapper around it.

Reply:
Actually, this will not only apply to the Ampath forms but also all the other forms like `startVisit Form`, `markDeceased`, `Allergies`, `vitals`, and all the other forms we have. That's my most challenge right now.

Reply:
It should be the responsibility of the component mounted into the workspace to tell you that... This code shouldn't be trying to deduce that at all.

Reply:
That's true, I have talked to some members of the team about including the caveat in a follow-up PR since it might need more discussions and critical thinking, so I will create a ticket for that work to capture updates from all the possible components that we mount in our workspaces. How does that sound @ibacher

Reply:
There was some discussion of this in the past. I think no one just got around to actually implementing it, so ticketing it sounds fine to me.